### PR TITLE
adding image-component

### PR DESCRIPTION
**Pull request by bipin7yadav**
-Round Image
-Responsive Image

### CODE REVIEW

File: image.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+.side-nav:hover{
+    background-color: var(--primary-color);
+    padding: 5px 3rem 5px 3rem;
+    border-radius: 1rem;
+}
+
+
+.sidebar{
+    width: 16rem;
+    height: fi;
```
Comment (mimansha-swarup) [inline image.css]: is there a typo?

File: image.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    position: relative;
+    margin-left: 4px;
+    width: 100%;
+}
+
+.body{
+    width: 100%;
+}
+.container{
+    padding: 3px;
```
Comment (mimansha-swarup) [inline image.css:18]: below you are using re for padding and here px
try to keep consistent

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

File: image.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    text-align: center;
+}
+
+.btn{
+    margin-top: 3rem;
+    display: flex;
+    justify-content: space-between;
+}
+
+button{
```
Comment (MarkVed17) [inline image.css:59]: Styling a semantic element is not good imo. Try using CSS selectors wherever possible.

File: image.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            <h3><a href="/Button componet/button.html" class="side-nav">Button</a></h3>
+            <h3><a href="/card component/card.html" class="side-nav">Card</a></h3>
+            <h3><a href="/img  comp/image.html" class="side-nav">Image</a></h3>
+            <h3><a href="/Input comp/input.html" class="side-nav">Input</a></h3>
+            <h3><a href="/list Component/list.html" class="side-nav">List</a></h3>
+            <h3><a href="/Navigation comp/navigation.html" class="side-nav">Navigation</a></h3>
+            <h3><a href="/rating comp/rating.html" class="side-nav">Rating</a></h3>
+            <h3><a href="/slider comp/slider.html" class="side-nav">Slider</a></h3>
+            <h3><a href="/text utilities/text-utility.html" class="side-nav">Text utilities</a></h3>
+
```
Comment (MarkVed17) [inline image.html:78]: These gaps between elements don’t look good. Try clearing it all before pushing changes.

File: image.html [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                class="carbon"
+                sandbox="allow-scripts allow-same-origin">
+              </iframe></div>
+
+            <div class="btn">
+                <button><a href="/card component/card.html">previous</a></button>
+                <button><a href="/Input comp/input.html">Next</a></button>
+               
+            </div>
+
```
Comment (MarkVed17) [inline image.html:117]: A gap here too.

**Review by MarkVed17 — APPROVED** [review]

File: image.html (hunk 4 above)
Comment (bipin7yadav) [reply image.html:78]: sure

File: image.css (hunk 2 above)
Comment (bipin7yadav) [reply image.css:18]: but vikram told us not rem in decimals

File: image.css (hunk 1 above)
Comment (bipin7yadav) [reply image.css]: no , i forgot to erase this

**Review by bipin7yadav — COMMENTED** [review]
made required changes

**Pull request merged**